solutions_automation/marketplace/meetings.py

```diff
@@ -0,0 +1,6 @@
+from jumpscale.packages.marketplace.chats.meetings import MeetingsDeploy
+from utils.gedispatch import GedisChatBotPatch
```

Comment:
please change this import to `from solutions_automation.utils.gedispatch import GedisChatBotPatch`